core/core_modules/admin.js

```diff
@@ -0,0 +1,176 @@
+﻿var config = require.once('../config.js'),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2587223488/issues/source?bid=3282987&fileBranchId=3292072#l1)